app/src/components/v-form/v-form.vue

```diff
@@ -43,6 +43,7 @@ const props = withDefaults(
 		direction?: string;
 		showDivider?: boolean;
 		inline?: boolean;
+		currentVersion?: ContentVersion | null;
```

Comment:
See the other comment

```suggestion
		version?: ContentVersion | null;
```